test/benchmarks/model_distillation.py

```diff
@@ -0,0 +1,99 @@
+from haystack.nodes import FARMReader
```

Comment:
FARMReader import is a duplicate here and `import os`is not needed.

Reply:
I removed the imports.